cfme/tests/infrastructure/test_host_role_type.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from utils import testgen
+from cfme.web_ui import Quadicon
+from cfme.web_ui import InfoBlock
+from cfme.fixtures import pytest_selenium as sel
+
+
+def pytest_generate_tests(metafunc):
+    argnames, argvalues, idlist = \
```

Comment:
This pattern has been discussed in multiple PRs, please fix it before we merge this PR.